ui/apps/platform/src/Containers/Clusters/ClustersPage.js

```diff
@@ -49,16 +49,6 @@ const ClustersPage = ({
     const { data: searchData } = useQuery(SEARCH_OPTIONS_QUERY, searchQueryOptions);
     const searchOptions = (searchData && searchData.searchOptions) || [];
 
-    // When the selected cluster changes, update the URL.
```

Comment:
This was existing UX weirdness that was made more apparent by this change. With this `useEffect` hook, history states would be pushed multiple times, requiring multiple "back clicks" to change the page URL. The addition of the new hook and removal of this chunk makes the URL and back button behave as expected.

Reply:
Oh, that is such good news! Thank you very much.